.travis.yml

```diff
@@ -1,14 +1,12 @@
 # This file was generated automatically from conda-smithy. To update this configuration,
-# update the conda-forge.yaml and/or the recipe/meta.yaml.
+# update the conda-forge.yml and/or the recipe/meta.yaml.
 
-language: objective-c
+language: generic
+
+os: osx
+osx_image: beta-xcode6.1
 
 env:
-  matrix:
-    
-    - CONDA_PY=27
-    - CONDA_PY=34
-    - CONDA_PY=35
```

Comment:
The other case where this seems to happen appears to be not having everything come from the `conda-forge` channel (if possible). That's what needed to happen for the re-rendering bot at least (see link below). Though I suspect this covering up the real problem.

xref: https://github.com/conda-forge/conda-forge-maintenance/pull/7